eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -46,18 +46,14 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
 
   inTransaction { pg =>
     using(pg.createStatement()) { statement =>
-      def migration45(statement: Statement): Int = {
+      def migration45(statement: Statement): Unit = {
         statement.executeUpdate("CREATE TABLE IF NOT EXISTS relayed_trampoline (payment_hash TEXT NOT NULL, amount_msat BIGINT NOT NULL, next_node_id TEXT NOT NULL, timestamp BIGINT NOT NULL)")
         statement.executeUpdate("CREATE INDEX IF NOT EXISTS relayed_trampoline_timestamp_idx ON relayed_trampoline(timestamp)")
         statement.executeUpdate("CREATE INDEX IF NOT EXISTS relayed_trampoline_payment_hash_idx ON relayed_trampoline(payment_hash)")
```

Comment:
Don't you want to remove the `IF NOT EXISTS`? If they do exist then it's a bug.

Reply:
Exactly, done here 945ada214. I didn't want to put SQL changes in the first commit.